content/en/docs/ibm/create-cluster.md

```diff
@@ -132,6 +158,124 @@ Wait until the cluster is deployed and configured. It can take a while for the c
 
 ```shell
 ibmcloud ks clusters --provider ${WORKER_NODE_PROVIDER} |grep ${CLUSTER_NAME}|awk '{print "Name:"$1"\tState:"$3}'
+```
+## For VPC infrastructure.
+
+Here we are assuming, that a user starts from scratch and creates a new vpc instance,
+but realistically a user may have an existing VPC with a public gateway already attached and also a subnet to create a cluster.
+Please feel free to directly proceed to `Step 5` and substitute appropriate values. 
+
+* Step 1. Install a vpc-infrastructure plugin.
+
+`$ ibmcloud plugin install vpc-infrastructure`
+
+* Step 2. Target the gen 2 to access gen 2 resources.
+
+`$ ibmcloud is target --gen 2`
+
+```$ ibmcloud is target
+Target Generation: 2
+```
+* Step 3. Create or use an existing VPC.
+
+List the available VPCs
+```shell
+ ibmcloud is vpcs
+Listing vpcs for generation 2 compute in all resource groups and region ...
+ID                                          Name                Status      Classic access   Default network ACL                                    Default security group                                 Resource group   
+r006-hidden-68cc-4d40-xxxx-4319fa3gxxxx   my-vpc1              available   false            husker-sloping-bee-resize                              blimp-hasty-unaware-overflow                           kubeflow   
+
+```
+So, if the above list contains the VPC that can be used to deploy your cluster note it's ID. Otherwise, proceed with creating a new VPC as follows.
+
+```shell
+$ ibmcloud is vpc-create my-vpc
+Creating vpc my-vpc in resource group kubeflow under account IBM as user prashsh1@in.ibm.com...
+                                                  
+ID                                             r006-hidden-68cc-4d40-xxxx-4319fa3fxxxx   
+Name                                           my-vpc   
+...  
+```
```

Comment:
You have a formatting issue here with Markdown not registering the end of the code block, which affects a bunch of lines. I suggest the following which may fix this (without indentation):

    
```
    ```shell
    ibmcloud is vpc-create my-vpc
    ```    

    Example output:

    ```
    Creating vpc my-vpc in resource group kubeflow under account IBM as user prashsh1@in.ibm.com...
                                                  
    ID                                             r006-hidden-68cc-4d40-xxxx-4319fa3fxxxx   
Name                                           my-vpc   
    ...  
    ```
```